docs/hardware/5-virtual-device/2-quickstart/5-simulating-devices-qemu.md

```diff
@@ -49,7 +50,7 @@ Save and exit (`ctrl+x` in nano)
 At this point, you can build a Golioth Zephyr project:
 
 ```
-west build -b qemu_cortex_m3 samples/hello -p
+west build -b qemu_cortex_m3 samples/hello -p -- -DOVERLAY_CONFIG=$HOME/device_creds/DEVICE_CRED_ID.conf

```

Comment:
> For QEMU device credentials, added an overlay config to the build command. This is not strictly needed, but it seems like more of a "best practice" for Zephyr, so you're not storing device-specific credentials and secrets in the project source tree.

There is a "best practice vs simplicity" involved here. We didn't recommend adding overlays before (but we were considering it), as it is another new magic thing that scares newcomers and is another mechanism that when used in a wrong way can be a pain.